pennylane/plugins/default_qubit.py

```diff
@@ -431,6 +452,15 @@ def _get_operator_matrix(self, operation, par):
         return A(*par)
 
     def ev(self, A, wires):
+        r"""Expectation value of observable on specified wires.
+
+         Args:
```

Comment:
args don't match function signature

Reply:
Corrected it. It's weird because I recovered this docstring from before #259 where it vanished. CodeFactor notified me that it was missing which is _weird_.